javascript/javascript2/week3/5.js

```diff
@@ -0,0 +1,31 @@
+//5. Log location
+
+function geoFindMe() {
+  const status = document.querySelector("#status");
+  const mapLink = document.querySelector("#map-link");
+
+  mapLink.href = "";
+  mapLink.textContent = "";
+
+  function success(position) {
+    const latitude = position.coords.latitude;
+    const longitude = position.coords.longitude;
+
+    status.textContent = "";
+    mapLink.href = `https://www.openstreetmap.org/#map=18/${latitude}/${longitude}`;
```

Comment:
could you also use:
https://wiki.openstreetmap.org/wiki/Export#Embeddable_HTML

:)